### ISSUE TITLE
Configure Buildkite pipelines

### ISSUE THREAD

**Issue by iampatbrown** [buildkite/test-collector-swift#2]
## 💬  Context

Some functionality in this library will vary depending on the platform and version of Swift. This can be challenging for a developer to test locally. If possible, we should set up test pipelines on macOS, Linux and Windows.

## 📝 Notes

- Supporting Linux is desirable and Windows would be lower priority
- macOS can execute tests via `xcodebuild test` or `swift test`
- Tests can be run in parallel using `swift test --parallel`

## 👩‍🔧 Technical Design Notes

I've attached the GitHub Actions I had previously been using. We can convert these to Buildkite pipelines.
[GitHub Actions.zip](https://github.com/buildkite/collector-swift/files/8794710/GitHub.Actions.zip)

**Comment by iampatbrown** [comment]
As mentioned in #20 
- [ ] We should add an `ObjC` example as well

**Comment by pda** [comment]
This repository has moved into the [buildkite/bktest](https://github.com/buildkite/bktest) monorepo — this project now lives at [`test-collector-swift/`](https://github.com/buildkite/bktest/tree/main/test-collector-swift), with full git history carried over.

As part of the move we're closing open issues here. If this still needs attention, please open a new issue in buildkite/bktest and link back to this one.
